### PR TITLE
added my contribution with right sizes and stuff

### PR DESCRIPTION
**Pull request by AvramPop**

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://AvramPop.github.io/gci15.fossasia.org

File: _data/gallery.yml [hunk 1]
```diff
@@ -162,4 +162,8 @@
     name: Call yourself in PHP
     image: configurePhpPiotr.png
     author: Piotr Kowalski
-
+    
+  -
+	name: YaCy page on ro.Wikipedia
```
Comment (jigyasa-grover) [inline _data/gallery.yml:167]: @AvramPop Follow the pattern. 
Remove the spaces/tab before `name:`

**Comment by abishekvashok** [comment]
Please fix as stated by @jig08 @AvramPop

**Comment by SaptakS** [comment]
Please fix the merge conflicts... https://github.com/fossasia/gci15.fossasia.org/blob/gh-pages/GIT.md

**Comment by AvramPop** [comment]
but ive corrected the mistake: d8d0776

**Comment by abishekvashok** [comment]
But there are conflicts common just resolve them or create a new PR with the updated code(re-clone and re-fork)

**Comment by jigyasa-grover** [comment]
@AvramPop The branch still has merge conflicts. Either follow the steps mentioned [here](https://github.com/fossasia/gci15.fossasia.org/blob/gh-pages/GIT.md) or if you wish you can start with a clean fork :) Let us know in case of any issues.

**Pull request closed without merging**